doc/programming_guide/tutorials/building_rtos_applications.rst

```diff
@@ -0,0 +1,11 @@
+##########################
+Building RTOS Applications
+##########################
+
+Applications using the RTOS Framework are built using `CMake`.
+The RTOS framework provides many libraries, drivers and software services, all of which can be included by the application's ``CMakeLists.txt`` file.
+The application's CMakeLists can specify precisely which drivers and software services within the SDK should be included through the use of various `CMake` target aliases.
+
+See the :ref:`Build System Guide <build_system_guide>` for more information on the build system.
```

Comment:
Reference to another document wont work on the pdf, we could point to the actual pdf cognidox link once is published, or just put the text on bold or mention where to find the document. Not sure about the best approach here but it can be a bit confusing for someone reading the pdf only. 

[fwk_rtos_programming_guide_v3.3.0.pdf](https://github.com/user-attachments/files/18098094/fwk_rtos_programming_guide_v3.3.0.pdf)

Reply:
I have replaced the references with **bold text**.